src/main/java/seedu/address/model/person/Volunteer.java

```diff
@@ -0,0 +1,88 @@
+package seedu.address.model.person;
+
+import java.util.Set;
+
+import seedu.address.model.person.information.Address;
+import seedu.address.model.person.information.Age;
+import seedu.address.model.person.information.Email;
+import seedu.address.model.person.information.Name;
+import seedu.address.model.person.information.Nric;
+import seedu.address.model.person.information.Phone;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents an Volunteer in the database.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Volunteer extends Person {
+    private final Nric nric;
+    private final Age age;
```

Comment:
If both `Volunteer` and `Elderly` have the `Nric` and `Age`, we should make them part of `Person`. What do you think?